### PR TITLE
RDM-6376- Functional test cases for "fetch a case for display" endpoint

### PR DESCRIPTION
**Pull request by aashishaggarwal123**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-6376

### Change description ###
Implement functional test cases for "fetch a case for display" endpoint

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/aat/resources/features/F-027/F-027.feature [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Feature: Fetch a case for display for Case Worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-038
+  Scenario: must return status 200 along with the case-view object successfully
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [has a valid case-reference in input parameters]
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-027/F-027.feature]: uncomment these lines please.

Comment (aashishaggarwal123) [reply src/aat/resources/features/F-027/F-027.feature]: The comments steps are enabled. please check and approve the PR.

File: src/aat/resources/features/F-027/S-035.td.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "vary" : "accept-encoding",
+      "X-Content-Type-Options" : "nosniff",
+      "Content-Encoding" : "gzip",
+      "Connection" : "close",
+      "Pragma" : "no-cache",
+      "Expires" : "0",
+      "X-XSS-Protection" : "1; mode=block",
+      "Date" : "[[ANY]]",
+      "Content-Type" : "application/json;charset=UTF-8"
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-027/S-035.td.json]: These should be inherited from base object

File: src/aat/resources/features/F-027/S-034.td.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "vary" : "accept-encoding",
+      "X-Content-Type-Options" : "nosniff",
+      "Content-Encoding" : "gzip",
+      "Pragma" : "no-cache",
+      "Expires" : "0",
+      "X-XSS-Protection" : "1; mode=block",
+      "Date" : "[[ANY]]",
+      "Content-Type" : "application/json;charset=UTF-8"
+    },
```
[the remark this answers is not in the document]
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-027/S-034.td.json]: Yes, I am implementing those changes. This PR was created before your update.

File: src/aat/resources/features/F-027/S-035.td.json (hunk 2 above)
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-027/S-035.td.json]: Changes done and updated.

File: src/aat/resources/features/F-027/S-034.td.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
+  "_guid_": "S-034",
+  "_extends_": "S-000-Base-Root",
```
[the remark this answers is not in the document]
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-027/S-034.td.json]: Changes for accommodating 6329 are added.

File: build.gradle [hunk 5]
```diff
@@ -478,9 +478,8 @@ jacoco {
 
 sonarqube {
     properties {
-        property "sonar.exclusions", "build/generated-sources/**/*.java,**/service/doclink/**"
-        property "sonar.coverage.exclusions", "src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java,src/main/java/uk/gov/hmcts/ccd/AppInsights.java," +
-                "**/DocLinksRestoreEndpoint.java,**/service/doclink/**"
+        property "sonar.exclusions", "build/generated-sources/**/*.java"
+        property "sonar.coverage.exclusions", "src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java,src/main/java/uk/gov/hmcts/ccd/AppInsights.java"
```
Comment (MSancaktutar) [inline build.gradle]: Please remove all the irrelevant changes from this PR.

**Comment by aashishaggarwal123** [comment]
New PR is created to replace this.

**Pull request closed without merging**